templates/search/blocks/featured-search__record-creator.html

```diff
@@ -1,28 +1,23 @@
 {% load search_tags %}
-{% load humanize %}
 <div class="featured-search__results-block">
-    <h2 class="featured-search__heading">{{ results.bucket.label }}</h2>
-    
-    {% if results.hits.hits %}
+    <h2 class="featured-search__heading">{{ bucket.label }}</h2>
 
-        {% with record=results.hits.hits.0 %}
+    {% for record in bucket.results|slice:":1" %}
```

Comment:
Quick Python know-how question: am I correct to interpret this use of `|slide:":1"` to mean just the first item?

Reply:
Yes, that's right. If we change the number of items we want to show at some point, this new format will be easier to change. It's worth bookmarking [this page of the Django docs](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/) if working a lot with Django. There are some very handy tidbits in there, including [slice](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#slice)